### DISCUSSION TITLE
Notes - 20th Feb 2024

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#11] [General]
## GDP Correlations Notebook Review

- https://github.com/QuantEcon/reading-group-2024/discussions/10

**Fig:** Nominal Vs. Real GDP

- re-index from 2017 to 1949 so the index year is at the start of the series, or
- cut the graph from 2017 onwards

Ways to count GDP

- [future] make the point that Expenditure, Income should come to the same value (maybe a circular graph to show the relationship between Banks, Households etc.)

## GDP Components

- bar chart graph of Components of GDP (good chart, we need to think of a representation so that to make the shares sum to 100 and have a more natural interpretation of the chart)

- move `GDP per capita` to x axis

## GDP and Gini Coefficient

- interesting graph, we may not use it in the lecture and focus the discussion on how to measure inequality and not bring in the growth and inequality discussion (due to complexity around that story)

## GDP + Energy

- show linear scale and log scale (with fit) vertically stacked as they have different `y` axis but the same `x` axis. 

## GDP + Child Mortality

- remove log scale to show the exponential relationship in the data (more naturally)

## GDP per Capita and Temperature Anomaly

- @mmcky to look at Angus Maddison for world GDP estimates (how long back can we go)
- See if we can find a longer time series for temperature anomaly and/or CO2 concentration

- [ ] push the latest notebook to the repo

---

## Inequality Lecture Review

- `5.1.1` explanation of what inequality is and why we care about it? maybe add a graph comparing inequality with happiness? @pgrosser1 to share link to data on reported happiness

- @mmcky to look at deprecation notice in the lecture

- `Lorenz Curve` should we use `qe` library or write our own function? Decision: move a version of Lorenz curve into the lecture rather than use the library so we can see how it is constructed and we can remove numba dependency for `WASM`

- (General) add labels to figures x and y axis

- `Fig 5.2` income is a flow and wealth is a stock (make that clear)

- maybe add some more Lorenz curve plots (across countries) or two ends of a long time period (within country) to see how it is evolving based on the GINI measure

- can we better connect the formulae with the graph (visually)? 

- `Fig 5.5` Why is the gini coefficient fluctuating like this in the US, Policy? other phenomena? Is this happening globally (a technology driver) or just a few countries (different policies)? Maybe talk about the lack of variance in the Gini Coefficient (due to how it is measured - area measure) with a narrow band of variation. (could pair with the discussion on shares)

- Compare West (US, UK) with Scandinavia (more social oriented policies) over the same period (let the data talk) -- while not drawing concrete conclusions but possibilities. 

**Idea:** Add a section on policies that target inequality (redistribution, income tax)?

**Comment by pgrosser1** [comment]
Apologies for the delay @mmcky but here is the dataset/s I found:
The Gini coefficient data is from here: https://data.worldbank.org/indicator/SI.POV.GINI?end=2022&most_recent_value_desc=true&start=2022&view=map&year=2015
The life satisfaction/self-reported happiness data is from here: https://ourworldindata.org/grapher/happiness-cantril-ladder?tab=table

I have combined all the countries that have data for both indicators into an Excel notebook:
[Happiness_Gini_allcountries.xlsx](https://github.com/QuantEcon/reading-group-2024/files/14380899/Happiness_Gini_allcountries.xlsx)
